lime/tools/helpers/CommandHelper.hx

```diff
@@ -0,0 +1,56 @@
+package lime.tools.helpers;
+
+
+import lime.project.CLICommand;
+import lime.project.Platform;
+
+
+class CommandHelper {
+	
+	
+	public static function executeCommands (commands:Array <CLICommand>):Void {
+		
+		for (c in commands) {
+			
+			Sys.command(c.command, c.args);
+			
+		}
+		
+	}
+	
+	public static function openFile (file:String):CLICommand {
```

Comment:
We already have `ProcessHelper.openFile` https://github.com/openfl/lime/blob/75bb6905af1b64b01af68892134afc00550846e0/lime/tools/helpers/ProcessHelper.hx#L21